deploy/manifests/quay-operator/3.4.0/quay-operator.clusterserviceversion.yaml

```diff
@@ -39,7 +40,7 @@ spec:
   customresourcedefinitions:
     owned:
     - description: Represents a full Quay registry installation.
-      displayName: Quay Registry
```

Comment:
The display names of CRDs should not change between upstream/downstream. The `kind` of the CRD is `QuayRegistry`, so it's display name should be "Quay Registry".